src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicSpecifyCultureForToLowerAndToUpper.Fixer.vb

```diff
@@ -21,25 +21,41 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
         End Function
 
         Protected Overrides Async Function SpecifyCurrentCultureAsync(document As Document, generator As SyntaxGenerator, root As SyntaxNode, node As SyntaxNode, cancellationToken As CancellationToken) As Task(Of Document)
-            If node.IsKind(SyntaxKind.IdentifierName) Then
-                Dim invocation = node.Parent?.FirstAncestorOrSelf(Of InvocationExpressionSyntax)
-                If invocation IsNot Nothing Then
-                    Dim model = Await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(False)
-                    Dim symbolInfo = model.GetSymbolInfo(node, cancellationToken).Symbol
-                    Dim methodSymbol = TryCast(symbolInfo, IMethodSymbol)
-
-                    If methodSymbol IsNot Nothing And methodSymbol.Parameters.Length = 0 Then
-                        Dim newArg = generator.Argument(CreateCurrentCultureMemberAccess(generator, model)).WithAdditionalAnnotations(Formatter.Annotation)
-                        Dim newInvocation = invocation.AddArgumentListArguments(DirectCast(newArg, ArgumentSyntax)).WithAdditionalAnnotations(Formatter.Annotation)
-                        Dim newRoot = root.ReplaceNode(invocation, newInvocation)
-                        Return document.WithSyntaxRoot(newRoot)
-                    End If
+            If ShouldFix(node) Then
+                Dim memberAccess = DirectCast(node.Parent, MemberAccessExpressionSyntax)
+
+                If memberAccess.Parent Is Nothing Or Not memberAccess.Parent.IsKind(SyntaxKind.InvocationExpression) Then
```

Comment:
@buyaa-n I think this needs to be `OrElse` too or there's a risk of null-ref exception.
I'm sorry I wasn't able to point this out in time until the branch was merged.

Reply:
@TKharaishvili there will be no null-ref exception. `IsKind` is an extension method and returns `false` if the receiver is null.

https://github.com/dotnet/roslyn/blob/9c4920a8d2538abd92d7e8880f556f75ad8a6588/src/Compilers/CSharp/Portable/CSharpExtensions.cs#L47-L50

Reply:
@Youssef1313 I see. Thanks for clarifying that.

Reply:
Thanks for pointing this out, that makes sense, no worries you can always create a new PR for fix

Reply:
Oh just saw @Youssef1313's reply, thanks clarification!